.lando.yml

```
@@ -63,19 +63,24 @@ tooling:
   install:
     service: appserver
     cmd:
-      - /app/vendor/bin/drush --root=/app/web site:install --account-mail=noreply@email.arizona.edu --account-name=azadmin --account-pass=azadmin --db-url=mysql://drupal10:drupal10@database:3306/drupal10 -y --verbose
-      - /app/vendor/bin/drush --root=/app/web config:set -y az_cas.settings disable_login_form 0
-      - /app/vendor/bin/drush --root=/app/web cache:rebuild
+      - /app/vendor/bin/drush site:install --account-mail=noreply@email.arizona.edu --account-name=azadmin --account-pass=azadmin --db-url=mysql://drupal10:drupal10@database:3306/drupal10 -y --verbose
+      - /app/vendor/bin/drush config:set -y az_cas.settings disable_login_form 0
+      - /app/vendor/bin/drush cache:rebuild
```

I'm thinking we shouldn't change anything with the existing drush commands tooling since folks might still be using Drush 11.
```suggestion
      - /app/vendor/bin/drush --root=/app/web site:install --account-mail=noreply@email.arizona.edu --account-name=azadmin --account-pass=azadmin --db-url=mysql://drupal10:drupal10@database:3306/drupal10 -y --verbose
      - /app/vendor/bin/drush --root=/app/web config:set -y az_cas.settings disable_login_form 0
      - /app/vendor/bin/drush --root=/app/web cache:rebuild
```